core/views.py

```diff
@@ -39,45 +39,11 @@ def password(request):
         else:
             generated_PASS = 'zaznacz cos wrr'
         return JsonResponse({'password': generated_PASS})
-  
-    
-def signup(request):
-    if request.method == 'POST':
-        username = request.POST.get('username')
-        email = request.POST.get('email')
-        pass1 = request.POST.get('pass1')
-        pass2 = request.POST.get('pass2')
-        myuser = User.objects.create_user(username, email, pass1)
-        myuser.save()
-
-        messages.success(request, 'Twoje konto zostało utworzone!')
-
-        return redirect('signin')
-
-    return render(request, 'core/signup.html')
-
-
-def signin(request):
-
-    if request.method == 'POST':
-        username = request.POST.get('username')
-        pass1 = request.POST.get('pass1')
-
-        user = authenticate(username=username, password=pass1)
-
-        if user is not None:
-            login(request, user)
-            messages.success(request, 'Zalogowano pomyślnie!')
-            return redirect('home')
-        else:
-            messages.error(request, 'Błędny login lub hasło!')
-            return redirect('signin')
-
-    return render(request, 'core/signin.html')
 
 
+class signup(CreateView):
```

Comment:
Zmień to na FormView, nazwa klasy z dużej litery - do nazywania klas używamy PascalCase.